### ISSUE TITLE
NotImplementedError: QgsPluginLayer.writeStyle() is abstract and must be overridden

### ISSUE THREAD

**Issue by drnextgis** [nextgis/quickmapservices#89]
В мастере при удалении слоя из списка слоёв:

```
An error has occurred while executing Python code: 

NotImplementedError: QgsPluginLayer.writeStyle() is abstract and must be overridden 
NotImplementedError: QgsPluginLayer.writeStyle() is abstract and must be overridden


Python version: 2.7.6 (default, Jun 22 2015, 18:01:27) [GCC 4.8.2] 
QGIS version: 2.15.0-Master Master, 4c56fa6 
```

**Comment by wonder-sk** [comment]
That was a temporary API problem introduced within 2.16 release and fixed before 2.16 got released. I suggest closing this issue.

**Comment by simgislab** [comment]
Martin, thanks for letting us know
